main.js

```diff
@@ -0,0 +1,69 @@
+const countryList = document.getElementById('countryList');
+
+async function getAllCountries() {
+    try {
+        const result = await axios.get('https://restcountries.eu/rest/v2/all');
+        const { data } = result;
+        data.sort((a, b) => {
+            return a.population - b.population;
+        });
+
+        data.map((country) => {
+            const { flag, name, region, population } = country;
+
+            const countryElement = document.createElement('li');
+            countryElement.setAttribute('class', 'country-clickable');
+
+            const flagElement = document.createElement('img');
+            flagElement.setAttribute('src', flag);
+            flagElement.setAttribute('class', 'flag');
+            // <img> aan ons <li> element toevoegen
+            countryElement.appendChild(flagElement);
+
+            const countryNameElement = document.createElement('span');
+            countryNameElement.textContent = name;
+            countryNameElement.setAttribute('class', getRegionClass(region));
+
+            countryElement.appendChild(countryNameElement);
+
+            const populationText = document.createElement('p');
+            populationText.setAttribute('class', 'population-dropdown');
+            populationText.textContent = `Has a population of ${population} people`;
+
+            countryElement.appendChild(populationText);
+           countryElement.addEventListener('click', () => {
+                toggleVisibility(populationText);
+            });
+            countryList.appendChild(countryElement);
+        });
+
+        console.log(data);
+    } catch(e) {
+        console.error(e);
+    }
+}
+
+getAllCountries();
+
+function toggleVisibility(populationElement) {
+    populationElement.classList.toggle('visible');
```

Comment:
Super, dat je .toggle gebruikt!

Reply:
Bedankt voor jouw peer review!

Was je nog in de gelegenheid geweest om naar onderstaande vraag te kijken: ik heb de website hiervoor aangemaakt in GitHub maar dan werkt de applicatie niet meer. De data laadt dan niet meer in. Terwijl de data wel inlaadt in webstorm als ik de pagina in de browser laadt. Via GitHub kreeg ik wel een melding o.a. over datalimiet!

Reply:
Hmm, ik heb zelf geen ervaring met het aanmaken van een domein van GitHub helaas! Ik kan je daarom ook geen antwoord geven... Probeer anders eens de vraag te stellen aan wat medestudenten. Je kan soms versteld staan de hoeveelheid kennis die bij studenten liggen 😉

Reply:
Hi Zwen, 
Bedankt voor jouw terugkoppleing
Dit is een functie onder settings waarbij je je applicatie kan publiceren. Bij Country part 1 had ik dit ook gedaan. Daar werkte die wel. Ik heb de vraag nu in het vragenkanaal bij mijn medecursisten uitgezet in frontend.
Groet Karina